demos/react/geo/src/Airports.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react'
+import { Marker, Popup, useMapEvents } from 'react-leaflet'
+import { DivIcon } from 'leaflet'
+import { useSearchBox } from 'react-instantsearch-hooks-web'
+import { useGeoSearch } from './useGeoSearch'
```

Comment:
unrelated to the PR, but we probably should at least expose this one in React InstantSearch Hooks too